### DISCUSSION TITLE
How to test chnges to library.json

### DISCUSSION THREAD

**Discussion by acaminiti** [andrew-codechimp/HA-Battery-Notes#706] [Q&A] [answered]
Hi @andrew-codechimp 
I have being trying to check if the devices that I added to battery-notes will work. But I have not being able to see them being detected on my HA "production" instance.

I do have enable battery notes in configuration.yaml:
![image](https://github.com/andrew-codechimp/HA-Battery-Notes/assets/56757107/3a04b1d0-a7b0-498f-af1b-3594bacd1f70)

But devices won´t show up. I think I have chosen the correct location to get the manufacturer and model: as you described in the docs:

_"The manufacturer and model should be exactly what is displayed on the Device screen within Home Assistant."_

To try to verify that my changes worked. proceed to uninstall the official battery notes from HACS and replaced it with my working copy with the added changes. When it reloads, it will only load all the devices as defined in the library.json, but for a reason decides to ignore the newly added ones.

You can see my added devices here: https://github.com/acaminiti/HA-Battery-Notes/commit/e48e174a73ad6faa5ed5a31aca020b3a972a459b

Wonder what am I doing wrong here? Thanks in advance.
Regards.

**Comment by andrew-codechimp** [comment, marked as answer]
The library gets overwritten with a download of the latest on a restart of HA. 
To test your own changes follow this previous answer on how to create a user library. 

https://github.com/andrew-codechimp/HA-Battery-Notes/discussions/536
